HW8_Collections_and_data_structures_Part 2.py

```diff
@@ -0,0 +1,52 @@
+# Required Task
+def uniq_elements_func(first_lst, second_lst):
+    uniq_elements = set(first_lst).symmetric_difference(set(second_lst))
+    return uniq_elements
+
+
+def same_elements_func(first_lst, second_lst):
+    same_elements = set(first_lst).intersection(set(second_lst))
+    return same_elements
+
+
+# Oprional Task
+def uppercase_func(elements):
+    uppercase_elements = elements.upper()
+    return uppercase_elements
+
+
+def numb_elements_func(elements):
+    if isinstance(elements, int):
+        return elements
```

Comment:
incorrect condition here, `filter` function should return `boolean` value but `element` is returned here
```suggestion
    return isinstance(element, int)
```